api/integrations/hg/zulip-changegroup.py

```diff
@@ -58,6 +60,7 @@ def format_summary_line(web_url, user, base, tip, branch, node):
         node=node[:12])
 
 def format_commit_lines(web_url, repo, base, tip):
+    # type: (str, Any, int, int) -> str
```

Comment:
Has there been any discussion with a mentor about the usage of `Any` here? You probably need to check the type of `repo` [here](https://www.mercurial-scm.org/wiki/MercurialApi).